src/overlays/actors/ovl_Obj_Comb/z_obj_comb.c

```diff
@@ -21,19 +31,178 @@ const ActorInit Obj_Comb_InitVars = {
     (ActorFunc)ObjComb_Update,
     (ActorFunc)ObjComb_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Comb/func_80B91B60.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Comb/func_80B91E4C.s")
+UNK_TYPE D_80B922E0[] = {
+    0x00000000, 0x00000000, 0x00000000, 0x4001FFFE, 0x00000000, 0x00010100, 0x00000000, 0x00000000, 0x000F0064,
+};
+
+UNK_TYPE D_80B92304[] = {
+    0x0A000909,
+    0x20000000,
+    0x00000001,
+    &D_80B922E0,
+};
+
+InitChainEntry initChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_CONTINUE),
+    ICHAIN_F32(unk_F4, 1100, ICHAIN_CONTINUE),
+    ICHAIN_F32(unk_F8, 100, ICHAIN_CONTINUE),
+    ICHAIN_F32(unk_FC, 900, ICHAIN_STOP),
+};
+
+extern UNK_TYPE D_050095B0;
+extern UNK_TYPE D_05009940;
+
+void ObjComb_Break(ObjComb* this, GlobalContext* globalCtx) {
+    Vec3f pos1;
+    Vec3f posSum;
+    Vec3f pos2;
+    Gfx** dlist = &D_05009940;
+    s16 scale;
+    s16 phi_s2 = 0;
```

Comment:
angle might be a more suitable name.
```suggestion
    s16 angle= 0;
```